source/policies/trustee-ballot-policy.rst

```diff
@@ -0,0 +1,43 @@
+Trustee Election Ballot Policy
+==============================
+
+Section 13.1 of our constitution requires that one-third of the elected trustees
+retire at each Annual General Meeting and that the members have the opportunity to
+fill the resulting vacancies (and others that may exist).
+
+In the event that there are the same number of candidates (or fewer) than vacancies
+available, then the election of those candidates shall be via `acclamation <https://en.wikipedia.org/wiki/Acclamation>`_
+
+If there are more candidates than vacancies, then an election via ballot will be carried
+out using `sequential proportional approval voting <https://en.wikipedia.org/wiki/Sequential_proportional_approval_voting>`_
```

Comment:
Is it worth spelling out what happens if there are *fewer* candidates than vacancies? Presumably the answer is just "We'll make sure that doesn't happen" but it might be worth discussing the process.

Reply:
L8